webapp/components/integrations/components/edit_command/index.js

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+import {getCustomTeamCommands, editCommand} from 'mattermost-redux/actions/integrations';
+import {getCommands} from 'mattermost-redux/selectors/entities/integrations';
+
+import EditCommand from './edit_command.jsx';
+
+function mapStateToProps(state, ownProps) {
+    const commandId = ownProps.location.query.id;
+
+    return {
+        ...ownProps,
+        commandId,
+        commands: Object.values(getCommands(state)),
```

Comment:
Redux store for commands won't change. Commands would be fetched into store on component mount, then i'm dealing only with internal component state. I don't see how my store could be changed when i'm still on EditCommand component, websocket isn't connected here.